cocos/core/scene-graph/base-node.ts

```diff
@@ -124,7 +124,7 @@ export class BaseNode extends CCObject implements ISchedulable {
      * @en Name of node.
      * @zh 该节点名称。
      */
-    @property
+    @editable
```

Comment:
Are these decorators needed?

Reply:
No, the properties on Node are fixed when they are used